docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -1159,27 +1159,31 @@ From the implementation above, tenants are resolved from the request path so tha
 
 [NOTE]
 ====
-If you also use xref:security-openid-connect-multitenancy.adoc[OIDC multitenancy] and both OIDC and Hibernate ORM tenant IDs are the same and must be extracted from the Vert.x `RoutingContext` then you can pass the tenant id from the OIDC Tenant Resolver to the Hibernate ORM Tenant Resolver as a `RoutingContext` attribute, for example:
+If you also use xref:security-openid-connect-multitenancy.adoc[OIDC multitenancy] and both OIDC and Hibernate ORM tenant IDs are the same,
+you can get the OIDC tenant id from the `OidcSession` CDI bean inside of the Hibernate ORM Tenant Resolver like in the example below:
```

Comment:
It does not work with bearer access tokens

Reply:
I am probably missing something, I 'll wait for answer below before I will address this comment. Thanks